2-mandatory/2-financial-times.js

```diff
@@ -4,34 +4,71 @@
     The home page of the web site has a headline section, which only has space for article titles which are 65 characters or less.
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
-function potentialHeadlines(allArticleTitles) {
-    // TODO
+function potentialHeadlines(allArticleTitles){
+    let articlelCharacters = [];
+        for(let title of allArticleTitles){
+        if(title.length <= 65){
+        articlelCharacters.push(title);
+    }
+    }
+        return articlelCharacters;
 }
-
```

Comment:
for Review 

function potentialHeadlines(allArticleTitles) {
    let headlines = [];

    for(let title of allArticleTitles) {
        if(title.length <= 65) {
            headlines.push(title);
        }
    }

    return headlines;
}